src/Collection.php

```diff
@@ -478,6 +480,23 @@ public function getNamespace()
     }
 
     /**
+     *
+     * @see Grop::__construct()
+     * @param mixed $keys Fields to group by. If an array or non-code object is passed, it will be the key used to group results.
+     * @param array $initial Initializes the aggregation result document.
+     * @param Javascript $reduce An aggregation function that operates on the documents during the grouping operation.
+     * @param array $options Command options
+     * @return Traversable
+     */
+    public function group($keys, array $initial, Javascript $reduce, array $options = [])
```

Comment:
This is also something to discuss with @derickr, but we may want to hold off on adding a `group()` helper in favor of promoting `aggregate` (the general suggestion for users) and `mapReduce` (which I feel still has a use, as can do everything `group` can do). I'd also like to double-check with our server team to check if `group` may be something they'd expect to deprecate in a future release.